examples/sample_config.json

```diff
@@ -10,14 +10,24 @@
       "reportOn values: name, size, diffs, checksum",
       "checksum_tool: e.g., md5,sha1sum",
       "ignore_prefix_for_ext: ignores common prefix for list of file extensions, list should match with diffs",
-      "default_comparison: default comparison type to perform",
+      "default_comparison: as  defined in reportOn",
+      "ignore_ext: ignore files with this extension between final '.' and end of name",
+      "guess_ext: Guess file type using unix 'file' command and match results with the output key to get extension",
       "If no diffs entry for a file extension compare based on: size, md5, sha1"
+
     ],
     "exitOn": ["diffs"],
     "reportOn": ["diffs"],
     "checksum_tool": "md5sum",
     "ignore_prefix_for_ext":[],
-    "debug": 0
+    "debug": 0,
+    "ignore_ext": [".gz.tbi", ".tbi", ".bam.bai", ".bam.csi", ".bam.crai"],
+    "guess_ext": {
+      "application/octet-stream": "NA",
+      "application/x-gzip": ".gz",
```

Comment:
Bad idea.  BAM is technically x-gzip.  I'd only attempt to do anything with files that come back as text/plain as there are many gz based files that are going to be binary data.